src/js/monkeys/mobile.js

```diff
@@ -20,6 +20,9 @@ mobile.generateHtml = function (data) {
   var $inner = $('<div />').appendTo($images)
     .addClass('landscape-images-inner');
 
+  var $letterContainer = $('#monkey')
```

Comment:
All monkeys on a page will need to mobile so they will already have the mobile class added